packages/@uppy/core/src/Uppy.js

```diff
@@ -7,41 +7,22 @@ const Translator = require('@uppy/utils/lib/Translator')
 const ee = require('namespace-emitter')
 const { nanoid } = require('nanoid/non-secure')
 const throttle = require('lodash.throttle')
-const prettierBytes = require('@transloadit/prettier-bytes')
-const match = require('mime-match')
 const DefaultStore = require('@uppy/store-default')
 const getFileType = require('@uppy/utils/lib/getFileType')
 const getFileNameAndExtension = require('@uppy/utils/lib/getFileNameAndExtension')
 const generateFileID = require('@uppy/utils/lib/generateFileID')
 const supportsUploadProgress = require('./supportsUploadProgress')
 const getFileName = require('./getFileName')
 const { justErrorsLogger, debugLogger } = require('./loggers')
+const {
+  Restricter,
+  defaultOptions: defaultRestrictionOptions,
```

Comment:
could the default options instead be declared per method inside the restricter module? that way core wouldn't need to have any knowledge about that object.

```js
checkRequiredMetaFieldsOnFile (file, requiredMetaFields = []) {
   ...
}
```

Reply:
I don't think this would work. The properties need to exist because other plugins, like dashboard, expect it in `opts`.